bot/handlers/administration/start_handler.py

```diff
@@ -37,19 +38,26 @@ def __init__(self, bot: Bot, logger: logging.Logger):
             "edycja": get_edycja_message,
             "edit": get_edycja_message,
             "zarzadzanie": get_zarzadzanie_message,
+            "zarządzanie": get_zarzadzanie_message,
             "management": get_zarzadzanie_message,
             "raportowanie": get_raportowanie_message,
             "reporting": get_raportowanie_message,
             "subskrypcje": get_subskrypcje_message,
             "subscriptions": get_subskrypcje_message,
             "skroty": get_shortcuts_message,
```

Comment:
stinky stinky stinky nazwy funkcji w polglish langzyku xD

Reply:
ale no mus żeby na polskie znaki te komendy nie były wrażliwe ale wsm może jest jakaś funkcja która po wpisaniu komendy by po prostu usuwała polskie znaki z message

Reply:
```
    async def _do_handle(self, message: Message) -> None:
        content = message.text.split()
        await self._log_system_message(logging.INFO, get_log_received_start_command(message.from_user.username, message.text))

        if len(content) == 1:
            await self.__send_message(message, get_basic_message())
        elif len(content) == 2:
            command = content[1].lower()   <----
            response_func = self.__RESPONSES.get(command)
            if response_func:
                await self.__send_message(message, response_func())
            else:
                await self.__send_message(message, get_invalid_command_message())
```

to co ładować tutaj coś w tym stylu żeby w tym dictcie były tylko wersje bez polskich znaków ?

```
import unicodedata

def remove_polish_chars(text):
    normalized_text = unicodedata.normalize('NFKD', text)
    return ''.join([char for char in normalized_text if not unicodedata.combining(char)])

# Przykład użycia
text = "ŻÓŁĆ"
cleaned_text = remove_polish_chars(text)
print(cleaned_text)  # zolc
```

Reply:
ale mi chodzi tylko o nazwy funkcji xD


`get_zarzadzanie_message` czyta sie okropnie, chodzilo mi jedynie zebys sie na jeden jezyk zdecydowal xD

Reply:
Aa takie buty ja założyłem że ci chodzi o te "duplikaty" w Dict i faktycznie może racja w tym jest. Pierwotnie myślałem o tym w ten sposób get_"nazwa_funkcjonalności" ale może faktycznie warto przełożyć.